text/src/autogluon/text/automm/configs/optimization/adamw.yaml

```diff
@@ -15,3 +15,4 @@ optimization:
   top_k: 3
   top_k_average_method: "greedy_soup"  # We support averaging method described in https://arxiv.org/pdf/2203.05482.pdf.
                                        # Currently support "uniform_soup", "greedy_soup", and "best".
+  efficient_finetune: null  # Can be 'bit_fit' (only finetune bias), 'norm_fit' (finetune the normalization terms + bias terms), or null
```

Comment:
We may need to add the comment: set `lr_decay`= 1.0 if using `bit_fit` or `norm_fit`.

Reply:
Yes, I will later work on a notebook about efficient finetuning. The plan is to also add adapter (potentially other methods) and compare a few options.